package byteback.test.exceptions

import byteback.annotations.Contract.*

class Exception2 @Return constructor() : Exception()
